src/stdio.jl

```diff
@@ -32,20 +32,41 @@ end
 const bufs = Dict{String,IOBuffer}()
 const stream_interval = 0.1
 const max_bytes = 10*1024
+# max output per code cell is 512 kb
+const max_output_per_request = 1 << 19
```

Comment:
Julia knows the type of a `const foo = Ref(bar)`; this is the advantage of `Ref` over a non-`const` global.